### ISSUE TITLE
twisted.names.client.createResolver's `servers` argument is incompletely documented

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4713]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4713|
|Type|defect|
|Created|2010-10-25 20:21:07Z|

The API documentation says this about it:

```
    @type servers: C{list} of C{(str, int)} or C{None}
    @param servers: If not C{None}, interpreted as a list of addresses of
    domain name servers to attempt to use.  Addresses should be in dotted-quad
    form.
```

What it leaves out is the fact that the int elements of the tuples in this list give the port number to contact the server on.

Attachments:

* [4713.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/885/8856067cf691809f0744ce9ecb0ea5a043439c79/4404e4665e181a4ea2df18a0a53d92a0b0967964.patch) (866 bytes) - added by michaelnt on 2011-01-23 10:51:34Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4713 4713
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__documentation__easy documentation, easy
time__1288038067000000 1288038067000000
changetime__1295796202000000 1295796202000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="michaelnt's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|michaelnt removed owner|
|-|-|

Attached a patch that add the requested information on port numbers

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks!  The copyright date on `client.py` needs to be updated, and there should be a news fragment describing this doc improvement.  These are simple, so I'll do them and apply.  Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [30568]) Add more API documentation for the `servers` parameter of `createResolver`

Author: michaelnt
Reviewer: exarkun
Fixes: [#4713](https://github.com/twisted/twisted/issues/4713)

Document the `int` element of the `tuple`s in the `servers` list as being the port number
to contact the server on.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
